libraries/src/MVC/View/ListView.php

```diff
@@ -162,7 +193,7 @@ public function display($tpl = null)
         }
 
         // Build toolbar
-        $this->addToolbar();
+        $this->addToolbar();        
```

Comment:
```suggestion
        $this->addToolbar();
```
Please revert this unnecessary change as white space at the end of a ljne violates code style rules.